homework-solutions/js/hw-5/task-2.js

```diff
@@ -18,4 +18,20 @@
  */
 let fizzBuzzResult = '';
 
+for(let i=1;i<=100;i++){
+if(i % 3 === 0 && i % 5 === 0 ){
+    fizzBuzzResult = fizzBuzzResult + i + ` - делится и на 3 и на 5` + '\n'; 
```

Comment:
Как вариант + используем шаблонньіе строки
```suggestion
    fizzBuzzResult += `${i} - делится и на 3 и на 5` + '\n'; 
```